splash/lua_modules/wraputils.lua

```diff
@@ -109,6 +109,31 @@ local function sets_callback(func, storage)
   end
 end
 
+local function pack_callback_return_value(func)
+  return function(cb, ...)
+    local mcb = cb
+
+    if type(cb) == 'function' then
+      mcb = function(...)
+        return { cb(...) }
+      end
+    end
+
+    return func(mcb, ...)
+  end
+end
```

Comment:
Here is that part
https://github.com/scrapinghub/splash/blob/master/splash/lua_runner.py#L176